### ISSUE TITLE
Adding amenity=place_of_worship to operator tree

### ISSUE THREAD

**Issue by arch0345** [osmlab/name-suggestion-index#5232]
This would be helpful for places of worship operated by a common party, such as The Church of Jesus Christ of Latter-day Saints and Kingdom Hall of Jehovah's Witnesses. Recommend to also add landuse=religious to the operator tree using the same operators. iD presets already include Kingdom Hall of Jehovah's Witnesses without any operator tags, so that should be removed if this were to be added to the index.

**Comment by 1ec5** [comment]
If I’m not mistaken, these denominations are tagged using `denomination`, not necessarily `operator`. (For LDS churches, I’ve been using `operator` to be more specific and indicate the ward if known.)

**Comment by arch0345** [comment]
That's true, most places of worship use denomination instead of operator. I've also been using `operator` for LDS churches. Would this mean adding a 'denomination' tree?

**Comment by 1ec5** [comment]
Maybe. Given [the recent discussion about `catholic=*`](https://lists.openstreetmap.org/pipermail/tagging/2021-August/thread.html#62194) on the tagging list, I’m not sure there’s any appetite for indiscriminate tagging of [`denomination:wikidata`](https://taginfo.openstreetmap.org/keys/denomination%3Awikidata). That key is used in some cases of very small, obscure denominations (often with long names unsuitable for keyword-style tagging), but these denominations would probably be too small to justify an entry in NSI (with the most common value currently having just 19 occurrences as of writing). Would this project be open to a tree that doesn’t use Wikidata at all but just serves as a more comprehensive, machine-readable copy of [the `denomination` wiki page](https://wiki.openstreetmap.org/wiki/Key:denomination)?

**Comment by bhousel** [comment]
> but these denominations would probably be too small to justify an entry in NSI (with the most common value currently having just 19 occurrences as of writing).

I've been adding small transit operators into NSI. (<10 instances are collected automatically, but even one-offs are welcome if it helps make the map better).  I've found that the size of the index really isn't an issue.

> Would this project be open to a tree that doesn’t use Wikidata at all but just serves as a more comprehensive, machine-readable copy of [the `denomination` wiki page](https://wiki.openstreetmap.org/wiki/Key:denomination)?

No, everything is built around wikidata right now, so I'd really prefer to create a new tree with a mainTag that's `something:wikidata=*`.

**Comment by 1ec5** [comment]
> I've been adding small transit operators into NSI. (<10 instances are collected automatically, but even one-offs are welcome if it helps make the map better). I've found that the size of the index really isn't an issue.

That makes sense for public transit operators, which are easy to filter by geography, but small denominations can often be scattered worldwide with, say, one or two places of worship per country. The size of the index may not be an issue, but scrolling through a lot of matches for, say, “[Church of Christ](https://en.wikipedia.org/wiki/Church_of_Christ)” or “[Church of Jesus Christ](https://en.wikipedia.org/wiki/Church_of_Jesus_Christ)” could be quite inconvenient unless NSI were to provide search result ranking hints to editors. Most denominations don’t have easily distinguishable logos, so one might have to do a bit of research to choose the right one.
